### PR TITLE
chore: add vscode settings to split prettier and biome

### PR DESCRIPTION
**Pull request by nazarhussain**
**Motivation**

Make sure the `prettier` and `biome` don't override each other. 

**Description**

The `biome` formatter works with JS/TS files only (will be extended later). So we keep prettier for rest of the file types. 

**Steps to test or reproduce**

- Experience in the VSCode.

### CODE REVIEW

File: lodestar.code-workspace [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "settings": {
+    "window.title": "${activeEditorShort}${separator}${rootName}${separator}${profileName}${separator}[${activeRepositoryBranchName}]${separator}[${activeRepositoryBranchName}]",
```
Comment (nazarhussain) [inline lodestar.code-workspace]: I personally like having this settings when I have multiple windows of VSCode open. Thought to share with everyone.

Comment (nflaig) [reply lodestar.code-workspace]: can you share a screenshot how this looks like

Comment (nazarhussain) [reply lodestar.code-workspace]: image

**Comment by nflaig** [comment]
should we fix the lint error in this PR?

**Comment by nazarhussain** [comment]
> should we fix the lint error in this PR?

Created one for it. https://github.com/ChainSafe/lodestar/pull/7152

**Review by nflaig — APPROVED** [review]

**Comment by wemeetagain** [comment]
:tada: This PR is included in [v1.23.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.23.0) :tada:

**Pull request merged**